.github/workflows/industrial_ci_action.yml

```diff
@@ -11,33 +11,70 @@ jobs:
       matrix:
         env:
           - {
-            ROS_DISTRO: foxy, ROS_REPO: main, CCOV_UPLOAD: true,
-            CMAKE_ARGS: "-DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_FLAGS='--coverage' -DCMAKE_CXX_FLAGS='--coverage'",
-            AFTER_RUN_TARGET_TEST: './.ci.prepare_codecov',
-            ADDITIONAL_DEBS: 'curl lcov grep'
+            IMAGE: 'foxy-ci',
+            CCOV: true,
+            AFTER_RUN_TARGET_TEST: './.ci.prepare_codecov'
+           }
+          - {
+            IMAGE: 'foxy-ci-testing',
           }
-          - {ROS_DISTRO: foxy, ROS_REPO: testing, CCOV_UPLOAD: false}
```

Comment:
Please don't mix in json syntax here (even though yaml is a superset of json). This reads much better (and is consistent) with the formatting of the remaining file:
```yaml
env:
  - ROS_DISTRO: ...
    ROS_REPO: ...
  - IMAGE: ...

```